libleptongui/src/color_edit_widget.c

```diff
@@ -378,6 +493,52 @@ on_color_sel_changed (GtkColorSelection* csel, gpointer p)
 
 } /* on_color_sel_changed() */
 
+#endif
+
+
+#ifdef ENABLE_GTK3
+/*! \brief "Apply" button "clicked" signal handler. */
+static void
+on_btn_apply (GtkWidget* btn, gpointer p)
+{
+  ColorEditWidget* widget = (ColorEditWidget*) p;
+
+  g_return_if_fail (widget != NULL);
+  g_return_if_fail (widget->toplevel_ != NULL);
+
+  GtkColorChooser* chooser = GTK_COLOR_CHOOSER (widget->color_chooser);
+  GdkRGBA color;
+  gtk_color_chooser_get_rgba (chooser, &color);
+
+  int color_index = x_colorcb_get_index (GTK_WIDGET (widget->color_cb_));
+  x_color_set_display_color (color_index, &color);
```

Comment:
Forgot to set outline color here?
`x_color_set_outline_color (color_index, &color);`

Reply:
Thank you for the point, my oversight :-)